src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -1152,6 +1152,20 @@ public static boolean isNotEmpty(final Object object) {
         return result;
     }
 
+    /**
+     * Get passed {@code object} in case it is not null, otherwise return the value produced by supplier
+     *
+     * Consider to utilize this method if lazy computation of default value makes sense in your case
+     *
+     * @param object to get if not null else to apply passed {@link Supplier}
+     * @param supplier {@link Supplier} to utilize in case provided {@code object} is null
+     * @param <T> the type of the object
+     * @return passed {@code object} in case it is not null, otherwise return the value produced by supplier
+     */
+    public static <T> T defaultIfNull(T object, Supplier<T> supplier) {
+        return object == null ? supplier.get() : object;
```

Comment:
This duplicates `ObjectUtils.getIfNull(T, Supplier<T>)`

Reply:
@Mikhail2048  right?

Reply:
I  am sorry, but I have not found any method in `ObjectUtils` with such signature.  There is `ObjectUtils.getIfNull(T, T)`  method, but it is not a lazy implementation

Reply:
See https://github.com/apache/commons-lang/blob/02f3bee7bce4bebb4e1bfe27b613a46adedccef6/src/main/java/org/apache/commons/lang3/ObjectUtils.java#L744

Reply:
@garydgregory Yeah, thats true)

Reply:
Yeah we can close this PR, it seems that this functionality is already present, thank you!